packages/shylu/shylu_node/tacho/src/Tacho_Util.hpp

```diff
@@ -30,6 +30,11 @@
 #include "mkl.h"
 #endif
 
+#if defined(KOKKOS_ENABLE_CUDA) 
+#include "cublas.h"
```

Comment:
@kyungjoo-kim is this safe to mix/match the two cublas version headers?

Reply:
@ndellingwood Exposing both header files to the same compilation unit caused a build error once in Sierra: https://github.com/trilinos/Trilinos/issues/1194 .

Reply:
Good catch. This compiles because I include both headers sequentially. So, all the v2 functions are available and the same core function names are redefined with v2 version functions. As long as I use v2 APIs in the code, it works. I will remove the cublas.h and v2 (new) version is the right one that we are supposed to use.

Reply:
@ndellingwood would you review again  ?

Reply:
From the comments here https://github.com/trilinos/Trilinos/commit/d47d1d09323fc3e7ab6feecdc673506503142976 it seems the removal of the "old" API (hiding away in cpp file) fixed compilation with applications using the newer cublas header; the recent commit removes the older header known to cause issues, @kyungjoo-kim comment says that a follow up PR will contain ETI changes to properly hide headers in cpp files.